CPS.md

```diff
@@ -285,19 +285,15 @@ Applicants are required to prove possession of the Private Key corresponding to
 
 ### 3.2.2 Authentication of organization and domain identity
 
-ISRG only issues Domain Validation (DV) certificates. When a certificate request includes a list of FQDNs in a SAN list, all domains in the list are fully validated prior to issuance.
+ISRG only issues Domain Validation (DV) certificates. All FQDNs listed in the certificate request are fully validated prior to issuance.
```

Comment:
Is it possible for a certificate request to somehow include a domain that would not be validated? Like, can an FQDN be included in any other field in a request? Seems like we should constrain this statement somehow?

Reply:
We could say "All FQDNs which will be listed in the certificate are fully...", maybe? That most closely matches the language from the BRs:
> The CA SHALL confirm that prior to issuance, the CA has validated each Fully‐Qualified Domain Name (FQDN) listed in the Certificate as follows:

Reply:
I like your suggestion. Let's do that.

Reply:
Done!